src/classes/CRLP_Batch_Base.cls

```diff
@@ -76,18 +76,33 @@ public abstract class CRLP_Batch_Base {
     /** @description The Summary Object Type this batch job will process */
     protected SObjectType summaryObjectType;
 
-    /** @description The Summary Object Type this batch job will process */
+    /** @description The Detail Object Type this batch job will process */
     protected SObjectType detailObjectType;
 
-    /** @description The Summary Object Type this batch job will process */
+    /** @description The Detail Object field that will be used in the inner join */
     protected String keyField;
 
+    /** @description Whether this class was constructed from within a scheduled job */
+    protected Boolean isScheduledJob = false;
+
+    /**
+     * @description Testing API to mock isScheduledJob value when running tests
+     * @param isScheduledJob Boolean value to which isScheduledJob will be set if called
+     * from within a running test
+     */
+    @TestVisible protected void setIsScheduledJob(Boolean isScheduledJob){
```

Comment:
Since this is only expected to be visible in tests, we could use `private` here instead of `protected`. The @TestVisible annotation should go on its own line.

Reply:
Ok, I'm going to put @TestVisible on its own line and switch to getter/setter format.  I tried to switch to Private but it actually needs to stay Protected (getter) since it is referenced from the extending classes (I'll make the setter explicitly private).